docker/frontend.Dockerfile

```diff
@@ -1,3 +1,3 @@
-FROM ghcr.io/navikt/pensjon-etterlatte/etterlatte-node-server:8ace6eeef06b458509b812f43c250cfbf9573a75
+FROM ghcr.io/navikt/pensjon-etterlatte/etterlatte-node-server:f5002765e4546012b1fa68a1865e98a8e5d40607
```

Comment:
Denne versjonen vil vel være utdatert så fort branchen merges til main og `etterlatte-node-server` bygges.

Reply:
Det kan fort hende. Kan dele den opp i 2 PRs for å være sikre på at vi får det nyeste bygget i prod